site/docs/master/csi.md

```diff
@@ -8,13 +8,28 @@ The following are the prerequisites for using Velero to take Container Storage I
 
  1. The cluster is Kubernetes version 1.17 or greater.
  1. The cluster is running a CSI driver capable of support volume snapshots at the [v1beta1 API level](https://kubernetes.io/blog/2019/12/09/kubernetes-1-17-feature-cis-volume-snapshot-beta/).
- 1. The Velero server is running with the `--features EnableCSI` feature flag to enable CSI logic in Velero's core. See [Enabling Features][1] for more information.
  1. The Velero [CSI plugin](https://github.com/vmware-tanzu/velero-plugin-for-csi/) is installed to integrate with the CSI volume snapshot APIs.
  1. When restoring CSI volumesnapshots across clusters, the name of the CSI driver in the destination cluster is the same as that on the source cluster to ensure cross cluster portability of CSI volumesnapshots
 
+# Installing Velero with CSI support
+
+Ensure that the Velero server is running with the `EnableCSI` feature flag. See [Enabling Features][1] for more information.
+Also, the Velero [CSI plugin][2] is necessary to integrate with the CSI volume snapshot APIs.
+
+Both of these can be added with the `velero install` command.
```

Comment:
it'd be nice to also describe/show how to enable the feature and add the plugin if the user already has velero installed (edit the deployment YAML to add `--features` to the args section, and run `velero plugin add` for the plugin)

Reply:
you could also simply link https://velero.io/docs/master/customize-installation/#enable-features